components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/user/core/hybrid/JdbcHybridRoleManager.java

```diff
@@ -319,7 +319,7 @@ public String[] getUserListOfHybridRole(String roleName) throws UserStoreExcepti
         }
     }
 
-    public String[] getPrimaryRoles() throws UserStoreException {
+    private String[] getPrimaryUsers() throws UserStoreException {
         String filter = HybridJDBCConstants.ALL_FILTER;
         List<String> lst = new LinkedList<String>();
         String sqlStmt = HybridJDBCConstants.GET_PRIMARY_USER_ROLES;
```

Comment:
Shall we rename the `GET_PRIMARY_USER_ROLES` constant as well?

Reply:
Will change. thanks